### ISSUE TITLE
Background colors do not render

### ISSUE THREAD

**Issue by northify** [torchlight-api/torchlight-laravel#35]
Hi, This is a strange thing but the theme background colors do not render. In this example I wrapped the Torchlight component in a div styled with Tailwind, bg red. The Torchlight bg is styled but does not show. All cache has been cleared several times with no luck.
![Screen Shot 2022-06-29 at 9 03 41 AM](https://user-images.githubusercontent.com/11839864/176483651-2f5db603-ca85-467e-975a-3bb47dde3660.png)

**Comment by northify** [comment]
The only way I can get the background to show is to add an "absolute" position on it.
![Screen Shot 2022-06-29 at 10 04 45 AM](https://user-images.githubusercontent.com/11839864/176494866-9920dacb-bae9-4e06-84b3-5f2c7f040340.png)

This is on a fresh install of laravel 9 and Tailwind 3

**Comment by dartunian** [comment]
I'm having this same issue, did you ever find a solution?

**Comment by dartunian** [comment]
@aarondfrancis Do you know of any fix for this?

**Comment by northify** [comment]
No, I did not find a solution for this. I moved on from this package and used something else.

**Comment by aarondfrancis** [comment]
You need to use the "basic css" from the docs, that should fix it! 

https://torchlight.dev/docs/css
